cartridge/failover/raft.lua

```diff
@@ -47,21 +53,28 @@ local function get_appointments(topology_cfg)
             local member = membership.get_member(server.uri)
 
             if member ~= nil
-            and member.payload.leader_uuid ~= nil
-            and member.payload.raft_term or 0 >= latest_term
```

Comment:
this line contains a priority bug of logical operations. It need to merge this pr asap.